lib/dealership.rb

```diff
@@ -0,0 +1,32 @@
+class Dealership
+  attr_reader :inventory,
+              :inventory_count
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(new_car)
+    @inventory << new_car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    @inventory_count > 1
```

Comment:
So what is this method wanting to know? 
- whether or not there is anything in our `@inventory` array.

To have inventory, we would need our `@inventory` array to be greater than 0.